### PR TITLE
Make games index

### PR DESCRIPTION
**Pull request by Grasscopper**

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -15,3 +15,21 @@
...
+    margin-top: 25px;
+}
+
+body {
+    background-color: #D1CEFF;
+}
+
+h4 {
+    text-align: center;
+}
```
Comment (safang99) [inline app/assets/stylesheets/application.css:35]: tab spacing of 4

File: app/controllers/api/v1/games_controller.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Api::V1::GamesController < ApplicationController
+    def index
+        render json: Game.all
+    end
+end
```
Comment (safang99) [inline app/controllers/api/v1/games_controller.rb:5]: tab spacing of 4

File: app/javascript/react/components/App.js [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
+    <BrowserRouter>
+      <Switch>
+        <Route exact path="/games" component={GamesIndexContainer}/>
+        <Route path="/" component={GamesIndexContainer}/>
+      </Switch>
+    </BrowserRouter>
+  )
 }
 
 export default App
```
Comment (safang99) [inline app/javascript/react/components/App.js:17]: tab spacing of 2 here which is ideal

File: app/models/game.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+class Game < ApplicationRecord
+    validates :title, presence: true
+
```
Comment (safang99) [inline app/models/game.rb:3]: lets remove this empty line

File: db/seeds.rb [hunk 5]
```diff
@@ -5,3 +5,10 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+
```
Comment (safang99) [inline db/seeds.rb:9]: duplicate empty lines so we can remove one of them

File: app/models/game.rb (hunk 4 above)
Comment (freerangeandy) [reply app/models/game.rb:3]: change will make it into the next PR

**Pull request merged**